packages/inula-adapter/examples/$forceUpdate/clock.jsx

```
@@ -0,0 +1,15 @@
+export default function (props) {
+  // useeffect and clearInterval operation mas no sense, but this is necessary
+  // to prevent problems between persistent vue components and temporary inula function components
+  useEffect(() => {
```

ESLint

[error] 'useEffect' is not defined.

(no-undef)